frontend/src/components/WorkoutTemplateCard/WorkoutTemplateCard.tsx

```diff
@@ -0,0 +1,33 @@
+import styles from "./WorkoutTemplateCard.module.css";
+
+// TODO: Change the types mentioned here to reflect final object
+type workoutTemplate = {
+  name: string;
+  exercises: Array<string>;
+};
+
+type WorkoutTemplateCardProps = {
+  template: workoutTemplate;
+};
+
+const WorkoutTemplateCard: React.FC<WorkoutTemplateCardProps> = ({
+  template,
+}) => {
+  // TODO: Implement onclick method to open a workout session page
+  return (
+    <div className={styles.container} onClick={() => {}}>
+      <p className={styles.cardTitle}>
+        {template ? template.name : "No name yet"}
```

Comment:
will the ``template`` prop ever be null?

Reply:
No template name will never be null. I did this because I didn't have any data to work with yet but I wanted to see some text to be able to style the template title. When I start working with data, I can replace this with just `{ template }` or if I somehow forget to replace it, the correct name will still be displayed always.